### PR TITLE
Dposv3/whitelist bonuses

### PR DESCRIPTION
**Pull request by gakonst**
- Adds LocktimeTier to candidate statistics and whitelist
- Adds CLI parameter.

Part of the backport of https://github.com/loomnetwork/loomchain/pull/959 (whitelist bonuses)

### CODE REVIEW

File: builtin/commands/dposV3_commands.go [hunk 1]
```diff
@@ -287,17 +286,25 @@ func WhitelistCandidateCmdV3() *cobra.Command {
...
+
+			tier := uint64(0)
+			if len(args) >= 3 {
+				tier, err = strconv.ParseUint(args[2], 10, 64)
+				if err != nil {
+					return err
+				}
+
+				if tier > 3 {
+					errors.New("Tier value must be integer 0 - 4")
```
Comment (aupiff) [inline builtin/commands/dposV3_commands.go]: typo, should say "0 - 3".

File: builtin/commands/dposV3_commands.go [hunk 2]
```diff
@@ -335,11 +342,25 @@ func ChangeWhitelistAmountCmdV3() *cobra.Command {
...
```
Comment (aupiff) [inline builtin/commands/dposV3_commands.go]: typo should say "0-3"

File: builtin/commands/dposV3_commands.go (hunk 1 above)
Comment (gakonst) [inline builtin/commands/dposV3_commands.go]:
```suggestion
					errors.New("Tier value must be integer 0 - 3")
```

File: builtin/commands/dposV3_commands.go (hunk 2 above)
Comment (gakonst) [reply builtin/commands/dposV3_commands.go]:
```suggestion
					errors.New("Tier value must be integer 0 - 3")
```

**Pull request merged**